clientgui/SkinManager.cpp

```diff
@@ -668,6 +682,18 @@ bool CSkinAdvanced::InitializeDelayedValidation() {
     m_iconApplicationIcon.SetDefaults(wxT("application"), wxT("boinc"));
     m_iconApplicationDisconnectedIcon.SetDefaults(wxT("application disconnected"), wxT("boincdisconnect"));
     m_iconApplicationSnoozeIcon.SetDefaults(wxT("application snooze"), wxT("boincsnooze"));
+#elif __APPLE__
```

Comment:
I'm not sure if this is the canonical way to check that we're running under macOS. If there's a better way, please let me know.